src/contexts/BorrowFormContext/BorrowFormContext.tsx

```diff
@@ -356,8 +367,21 @@ export const BorrowFormProvider: React.FunctionComponent<BorrowFormProviderProps
         });
       }
     }
+  };
+
+  useEffect(() => {
+    void selectedFixedDebtChanged();
```

Comment:
why is this called using void?

Reply:
saw it in other places in the file and thought we want to stick to this. However I'm not sure who introduced this in the first place anx whether we do want to stick to this. @ssbarbee any thoughts on this?

Reply:
The void is used when you don't want to await an async function. In this context it doesn't make sense. I think you can remove the void. 

It resolves no-floating-promises because it does something (or rather, explicitly nothing) with the promise.